starter/source/loads/pblast/hm_read_pblast.F

```diff
@@ -361,12 +462,20 @@ SUBROUTINE HM_READ_PBLAST(ITAB    ,ITABM1    ,UNITAB   ,IGRSURF  ,NUMLOADP,
                   ENDIF                                                                
                 ENDDO                                                                  
                 ! Provided ID is referring to an nonexistent surface                                                     
-                IF (.NOT.lFOUND) THEN                                                  
-                  CALL ANCMSG(MSGID=73,ANMODE=ANINFO,MSGTYPE=MSGERROR,                 
-     .                        I1=ID,                                                   
-     .                        C1=TITR,                                                 
-     .                        I2= SURF_ID_GROUND,                                                 
-     .                        C2='DOES NOT REFER TO A VALID /SURF ID')                 
+                IF (.NOT.lFOUND) THEN 
```

Comment:
Hi @Schwoertzig, 
Maybe the code that is commented out could be remove?

Reply:
Thank you @laurent-altr for your feedback. I am removing these commented lines.